src/components/SocialLink.jsx

```diff
@@ -5,7 +5,7 @@ import PropTypes from "prop-types";
 const SocialLink = ({ icon, href, text }) => {
   return (
     <div>
-      <Link to={href}>
+      <Link to={href} target={"_blank"}>
```

Comment:
We use `{}` when we want to use jsx mainly when using variables.
In this case it can be simply `target="_blank"`

Reply:
Fixed